build/web-client-rpc.md

```diff
@@ -0,0 +1,55 @@
+# Web Client VS RPC Client
+
+If you're unsure which client to use for interacting with the Nimiq blockchain, this page offers a comparison between the Nimiq Web Client and the RPC so you can choose the tool that best fits your needs and expertise. {.subline}
+
+### TL;DR
+The Nimiq Web Client establishes direct connections with other peers in the network, making it a part of the blockchain network. In contrast, the RPC follows a client-server model, where clients connect to a single server that interacts with multiple peers on the network.
```

Comment:
I'd switch the order of either the mentions of the client types in the text, or the order in the graphic. Right now the web client is mentioned first, but the RPC client is on the left, which confused me at first.

Reply:
For me, an important differentiating factor is that the web-client only runs in Javascript environments, whereas RPC can be used from any programming language and environment, as it's _just_ HTTP calls.